portal/src/OnboardingSurveyScreen.tsx

```diff
@@ -600,22 +600,24 @@ function Step3Team(_props: StepProps) {
             selectedChoices={companySizeChoicesState}
             setChoice={setCompanySizeChoicesState}
           />
-          <PhoneTextField
-            label={renderToString(
-              "OnboardingSurveyScreen.step3-team.phone.label"
-            )}
-            inputValue={companyPhone.rawInputValue}
-            initialCountry={geoIPCountryCode ?? undefined}
-            onChange={(v) => setCompanyPhone(v)}
-          />
+          {!viewerLoading && viewer != null ? (
+            <PhoneTextField
+              label={renderToString(
+                "OnboardingSurveyScreen.step3-team.phone.label"
+              )}
+              inputValue={companyPhone.rawInputValue}
+              initialCountry={viewer.geoIPCountryCode ?? undefined}
+              onChange={(v) => setCompanyPhone(v)}
+            />
+          ) : null}
         </FormProvider>
       </div>
     </SurveyLayout>
   );
 }
 
 function Step3Personal(_props: StepProps) {
-  const { viewer } = useViewerQuery();
+  const { viewer, loading: viewerLoading } = useViewerQuery();
```

Comment:
We should wait until this query is fully loaded before showing the whole screen, like we did in other screens.

Reply:
I made it like this because I didn't find a loading state for this screen. Maybe I just add a loading spinner.